cypress/README.md

```diff
@@ -35,6 +35,24 @@ See best [the docs on best practice for selectors](https://docs.cypress.io/guide
 
 Follow [Cypress best practice](https://docs.cypress.io/guides/references/best-practices)
 
+### How to use cypress-autorecord
+To allow for auto-recording and stubbing to work, require `cypress-autorecord` in each of your test file and call the function at the beginning of your parent describe block.
+
+```js
+const autoRecord = require('cypress-autorecord'); // Require the autorecord function
+  
+describe('Home Page', function() { // Do not use arrow functions
+  autoRecord(); // Call the autoRecord function at the beginning of your describe block
```

Comment:
We don't actually want to call this every time - only initially to get the mocks and commit them to version control.